index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>BridgeMap - coming soon page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="An online directory for young people aged between 10-25 in England that lists organisations which offer free resources and support.">
+  <meta name=”robots” content=”index, follow”>
+  <meta property="og:type" content="Coming soon" />
+  <meta property="og:title" content="BridgeMap" />
+  <meta property="og:description" content="An online directory for young people aged between 10-25 in England that lists organisations which offer free resources and support." />
+  <meta property="og:image" content="assets/logo.jpg" />
+  <meta property="og:url" content="https://bridgemap.org" />
+  <meta property="og:site_name" content="BrideMap" />
+  <link rel="stylesheet" href="styles/reset.css">
+  <link rel="stylesheet" href="styles/index.css">
+</head>
+<body>
+  <main class="main">
+    <img src="assets/logo.svg" alt="BridgeMap logo" class="logo"/>
+    <h1 class="coming-soon-text">Coming soon</h1>
+    <p class="slogan-text">An online directory for young people aged between 10-25 in England that lists organisations which offer free resources and support.</p>
+    <div class="links-container">
+      <a href="/founders-note.html" class="links">
+        Founder’s note &rarr;
+      </a>
+      <a href="/founders-note.html" class="links">
+        Let us know about an organisation &rarr;
+      </a>
+      <a href="/founders-note.html" class="links">
+        How you can help? &rarr;
+      </a>
+    </div>
+    <div class="social-links-container">
+      <a href="/founders-note.html" class="social-links">
+        <img src="assets/social -icons/facebook.svg"/>
```

Comment:
i think we should remove the spacing in `social- icons` to be just `social-icons`

Reply:
done